packages/gcf-utils/package.json

```diff
@@ -42,11 +43,11 @@
     "c8": "^7.1.0",
     "cross-env": "^7.0.0",
     "dotenv": "^8.0.0",
-    "gts": "^1.1.2",
+    "gts": "^2.0.0-alpha.8",
```

Comment:
Would it be possible to de-tangle the TypeScript/gts upgrades from probot?  If we need to roll back ....

Reply:
If. we end up needing to rollback, I'll make a hot fix for the couple changes that were necessary in `release-please`. I'm not concerned about the probot upgrade, so much as the octokit update, which is only in one place.